vcs/VTKPlots.py

```diff
@@ -423,7 +418,13 @@ def clear(self, render=True):
     def createDefaultInteractor(self, ren=None):
         defaultInteractor = self.renWin.GetInteractor()
         if defaultInteractor is None:
-            defaultInteractor = vtk.vtkRenderWindowInteractor()
+            if self.bg:
```

Comment:
looks good @danlipsa 

quick question what happens if we change the bg=xx multiple times..does this function gets called every time

Reply:
The function is called only if the render window is not None so it won't be called every time.

Reply:
@aashish24 good p[oint. @danlipsa should we (in a spearate PR) lock self.bg once something has been plotted?